File: screenshot.js
const { chromium } = require('playwright');
const path = require('path');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();

  const reportPath = path.resolve('coverage/lcov-report/index.html');
  await page.goto(`file://${reportPath}`);

  await page.screenshot({ path: 'baseline-coverage.png', fullPage: true });

  await browser.close();
})();

File: screenshot.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    page = browser.new_page()
    page.goto('http://localhost:5173/')
    page.wait_for_timeout(3000)
    page.screenshot(path='settings-after.png')
    browser.close()
